src/config/enhance-octokit.ts

```diff
@@ -40,8 +45,7 @@ const instrumentRequests = (octokit: GitHubAPI) => {
  * (Because Probot instantiates the Octokit client for us, we can't use plugins.)
  */
 export default (octokit: GitHubAPI): GitHubAPI => {
-
-	OctokitError.wrapRequestErrors(octokit);
+	// OctokitError.wrapRequestErrors(octokit);
```

Comment:
Should we delete the line if we don't need it?  (Otherwise comment needed explaining why we commented it)

Reply:
done.